AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -212,75 +213,22 @@ + (Class)layerClass
 
 + (void)scheduleNodeForRecursiveDisplay:(ASDisplayNode *)node
```

Comment:
Considering this file is so large and already needs some further subdivision, removing this complexity from here is a fantastic win.